### ISSUE TITLE
infinite grind getting exhausting, unmodifiable as server OP

### ISSUE THREAD

**Issue by tacticalskirt** [AppliedEnergistics/Applied-Energistics-2#5339]
right, so we're using INDREV as our companion mod, on the fabric edition
nothing on indrev will make the quartz dusts, you need to turn that crank manually
its AWFUL! its boring its slow its bad!
so naturally i tried turning it up?
oh i cant, like Actually do that?
well i can modify the ore double percent chance, to like 250?
oh that hardcrashes on config load?

like i cannot emphesise, a CHANCE to double DUST is BAD
it is Bad Gameplay and its genuinely driving players off my server
either let me modify quartz - dust ratios, or include a powerable quartz crusher
or, ya know, whatch as i make a mod thats literally less than 50 lines of code to add a quartz crusher to your mod, and Literally everyone who downloads your mod downloads it
like, genuinely Every Player has commented that "it seems like a massive oversight that the mod about automation is, fudementally, unautomatable"
like no matter what you do if you're building a Large project? you're gonna be grinding at that stone for Literal Consecutive Hours.
fucked up and wrong.
bad~~!
like i dont mean to be cruel here but What? the fuck? were you THINKIN?? like, did you even PLAY in survival mode before releasing it?

**Comment by Pingger** [comment]
TL/DR: If you want to see your issues resolved: be polite and include ALL necessary information.

Seriously, get a grip on your attitude.
AE2 at its core is a STORAGE Mod, that allows you to automate normal crafting. And it also got the feature to interface with other machines, to allow more complex crafting with machines from other mods.

If you actually want to automate grinding, you should maybe include a mod that brings a more automatable grinding machine?!
AE2 is designed to be used in conjunction with other mods. AFAIK the included manual Grinder essentially exists to allow you access to AE2 stuff in the VERY early game.

Any maybe ... just maybe, if you actually had read the **issue template**, where, for a bug, you are required to specify the exact version used, or, for a feature request, a post structure, that allows easy reading.

You mentioned crashes: **Where is the corresponding crashreport?**

> or, ya know, whatch as i make a mod thats literally less than 50 lines of code to add a quartz crusher to your mod, and Literally everyone who downloads your mod downloads it

Sure, make that mod. But I guarantee, that most players, won't actually use it, as most either include EnderIO, IndustrialCraft or ThermalFoundation/Expansion/..., which all bring a powered, easily automatable grinder/crusher with them. You don't need to invent the wheel anew in each mod!

Ohh and if you can make it to actually be in less than 50 lines of code, send me a link. Keep in mind, that Java Coding standard specifies a 80 character limit per line, where basically each semi-colon is to be treated as the end of a line (among other things, and some exceptions like in the head of a for-loop)


And in response to #5340: THIS IS NOT REDDIT OR TWITTER OR FACEBOOK. If you want to rant, then go there and see if anyone cares.

Some other things:
What is INDREV?! I only know INDEV, or do you mean "Industrial Revolution" for Fabric? In case of "Industrial Revolution", that fabric mod is in Alpha, so not even close to be used in conjunction with other mods. They probably will also add some automated grinder/crusher in their mod at some point, since the Forge-Version has one. Maybe add a feature request there?

> nothing on indrev will make the quartz dusts, you need to turn that crank manually
> its AWFUL! its boring its slow its bad!
> so naturally i tried turning it up?
> oh i cant, like Actually do that?


You can change it: https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/master/src/main/resources/data/appliedenergistics2/recipes/grinder/certus_quartz_dust.json
But because of your rudeness I won't explain further

> well i can modify the ore double percent chance, to like 250?
> oh that hardcrashes on config load?

Clearly you are doing something wrong. Also ... crashreport?

> it is Bad Gameplay and its genuinely driving players off my server
> either let me modify quartz - dust ratios, or include a powerable quartz crusher

cool, but why would we care. It's not like you're paying the Devs to work on the mod are you? Instead you ranting about them, when they have exceptional work over the years.

> or, ya know, whatch as i make a mod thats literally less than 50 lines of code to add a quartz crusher to your mod, and Literally everyone who downloads your mod downloads it
see comment of this further up

> like, genuinely Every Player has commented that "it seems like a massive oversight that the mod about automation is, fudementally, unautomatable"

see comment on this further up

> like, genuinely Every Player has commented that "it seems like a massive oversight that the mod about automation is, fudementally, unautomatable"

Press 'X' to doubt. XXXXXXXXXXXXXXXXXXXXXXX...

> like no matter what you do if you're building a Large project? you're gonna be grinding at that stone for Literal Consecutive Hours.
> fucked up and wrong.
> bad~~!
> like i dont mean to be cruel here but What? the fuck? were you THINKIN?? like, did you even PLAY in survival mode before releasing it?

The devs or offering this mod voluntarily and work on it their own free time ... They are not getting paid!
So if you actually want the issue to be resolved:
1. Read the Github TOS and referenced Documents: https://docs.github.com/en/github/site-policy/github-terms-of-service
1. Specifically read the Community Guidelines, which were part of 1., but I want to make sure: https://docs.github.com/en/github/site-policy/github-community-guidelines
1. Be polite
1. Actually read the issue template and provide the requested information
1. Be prepared to answer some issue specific questions, that will probably arise from your specific issue

Keep in mind, that you will probably not receive a response, if you are rude (also keep in mind, that _I_ would classify this issue, as much worse than just "rude")
Also keep in mind, that both your issues #5339 and #5340 are quite possibly enough to actually cause sanctions on your github account if it was reported to GitHub and not handled _this_ conservatively by the Repository Contributors

Still:
Have a nice day

**Comment by Pingger** [comment]
PS: I am not sure if this is a private modpack, but should take a look at this: #5183
